com.unity.render-pipelines.universal/Runtime/Passes/AdditionalLightsShadowCasterPass.cs

```diff
@@ -865,6 +865,7 @@ void SetEmptyAdditionalShadowmapAtlas(ref ScriptableRenderContext context)
         {
             CommandBuffer cmd = CommandBufferPool.Get();
             CoreUtils.SetKeyword(cmd, ShaderKeywordStrings.AdditionalLightShadows, true);
+            cmd.SetGlobalTexture(m_AdditionalLightsShadowmap.id, m_AdditionalLightsShadowmapTexture);
```

Comment:
I supposed this might be obsolete with RTHandles PR? can you check?

Reply:
I will check, but in any case I think we need to land this regardless the rt handles